src/pages/home/ReportScreen.js

```diff
@@ -188,10 +188,6 @@ class ReportScreen extends React.Component {
     }
 
     render() {
-        if (!this.props.isSidebarLoaded || _.isEmpty(this.props.personalDetails)) {
-            return null;
-        }
-
         if (ReportUtils.isDefaultRoom(this.props.report) && !ReportUtils.canSeeDefaultRoom(this.props.report, this.props.policies, this.props.betas)) {
             return null;
```

Comment:
Hmm, I am not sure with that because maybe there is some case the user can't see the room, thus we just return null instead of showing a skeleton loader.

Reply:
Maybe, when a user has given an access to a policy room and then they create a new room. After a while, the access is revoked and when he tries to view it again, they will see nothing. If they see a skeleton loader, they will think that their chat is loading forever.

I don't know though whether that will happen or not, but I am sure returning null is better instead of showing a skeleton loader. (if we have a no access view, that would be better)

Reply:
Just remember one thing, when the access is revoked, the room will be filtered out from the LHN, but they can still access it directly through the browser search bar.

Reply:
Maybe those 2 if conditions that makes some blank chat issues being reported multiple times on Slack. 

One of the example https://app.slack.com/client/T03SC9DTT/C049HHMV9SM